agent/custom/general_navigation_action.py

```
@@ -624,306 +354,425 @@ def _iou(a, b):
+
+                # 无 t(识别不到任何带坐标关卡) -> 空屏计数, 超限失败; 否则盲滑
+                if t_xy is None:
+                    empty_count += 1
+                    mfaalog.info(f"[导航] 空屏(无坐标锚点) {empty_count}/{EMPTY_SCREEN_LIMIT}")
                     if empty_count >= EMPTY_SCREEN_LIMIT:
-                        mfaalog.info(f"[导航] 连续 {empty_count} 屏未识别到关卡, 判定到底")
-                        break
+                        mfaalog.info("[导航] 连续空屏达阈值, 无法导航(无兜底)")
+                        return CustomAction.RunResult(success=False)
+                    if last_dir is not None:
+                        start = (int(round(SCREEN_CENTER[0] - SWIPE_SIGN * last_dir[0] * SWIPE_DIST)),
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)